app/models/fee/base_fee.rb

```diff
@@ -136,7 +136,7 @@ def perform_validation?
 
     # where fee type not available, default calculated? to true
     def calculated?
-      fee_type&.calculated?.nil? ? true : fee_type.calculated?
+      fee_type&.calculated?.nil? || fee_type.calculated?
```

Comment:
I had considered this instead (and likewise in `base_claim.rb`):

```suggestion
      return true if fee_type&.calculated?.nil?
      
      fee_type.calculated?
```